packages/dashboard/src/portal/components/FarmNodesTable.vue

```diff
@@ -629,6 +686,27 @@ export default class FarmNodesTable extends Vue {
     }
     this.openPublicConfigDialog = true;
   }
+
+  async openExtraFee(node: nodeInterface) {
+    this.nodeToEdit = node;
+    // convert fees from USD to mili USD while getting
+    this.extraFee =
+      (await this.queryClient.contracts.getDedicatedNodeExtraFee({ nodeId: this.nodeToEdit.nodeId })) / 1000;
```

Comment:
the value should be multiplied by 1000 not divided

Reply:
thought it was on setting not the getting, so this should be ignored

Reply:
It's divided while getting & multiplied while setting.